protocol/x/sending/types/message_withdraw_from_subaccount.go

```diff
@@ -26,16 +26,6 @@ func NewMsgWithdrawFromSubaccount(
 	}
 }
 
-// GetSigners specifies that the sender of the message must sign.
```

Comment:
For my understanding, going forward how does this message indicate that `msg.Sender.Owner` is the expected signer?